spec/tictactoe_clojure/game_setup_spec.clj

```diff
@@ -0,0 +1,40 @@
+(ns tictactoe-clojure.game-setup-spec
+  (:require [speclj.core :refer :all]
+            [tictactoe-clojure.game-setup :refer :all]
+            [tictactoe-clojure.console :as console]
+            [tictactoe-clojure.player :as player]))
+
+(describe "tictactoe-clojure.game-setup"
+  (with human-player-X (player/create-player "Human" "X"))
+  (with human-player-O (player/create-player "Human" "O"))
+  (with computer-player-O (player/create-player "Computer" "O"))
+
+  (describe "#get-board-size"
+    (it "returns 3 if the user selects a 3x3 board"
+      (with-out-str (with-in-str "3"
+        (should= 3 (get-board-size)))))
+    (it "returns 4 if the user selects a 4x4 board"
+      (with-out-str (with-in-str "4"
+        (should= 4 (get-board-size)))))
+    (it "returns 5 if the user selects a 5x5 board"
+      (with-out-str (with-in-str "5"
+        (should= 5 (get-board-size))))))
```

Comment:
For all of these things that have recursion, how come you didn't test for the non-valid case?